tests/upgrade/test_crossgrade.sh

```diff
@@ -129,13 +130,14 @@ withRetries() {
     shift
     shift
     while (( n < max_retries )); do
-      echo "RUNNING $*" ; "${@}" && break
+      echo "RUNNING $*" ; "${@}" > "${ERROR_TMP_LOG}" 2>&1 && break
       echo "Failed, sleeping ${sleep_sec} seconds and retrying..."
+      cat "${ERROR_TMP_LOG}"
       ((n++))
       sleep "${sleep_sec}"
     done
 
-    if (( n == max_retries )); then die "$* failed after retrying ${max_retries} times."; fi
+    if (( n == max_retries )); then cat "${ERROR_TMP_LOG}"; die "$* failed after retrying ${max_retries} times."; fi
```

Comment:
an then you don't need to cat the log file

Reply:
i could do it but i think outputting the error here will give more context about the error. wdyt?